stan/fit.py

```diff
@@ -127,6 +127,9 @@ def __getitem__(self, param):
         num_samples_saved = (self.num_samples + self.num_warmup * self.save_warmup) // self.num_thin
         assert self._draws.shape == (len(self.sample_and_sampler_param_names) + len(self.constrained_param_names), num_samples_saved, self.num_chains)
         # fmt: on
+        if not len(param_indexes):
+            # return (empty) array with atleast one 0-dimension
+            return np.empty(param_dim + [num_samples_saved * self.num_chains])
```

Comment:
```suggestion
            assert 0 in param_dim
            return np.array([]).reshape(param_dim + [num_samples_saved * self.num_chains])
```

I prefer this because `np.empty` has strong connotations of allocating space, just like `np.ones` and `np.zeros`. Also there is a confusion between two senses of the word `empty`.

Reply:
That solution looks good too. I can change the code.